pkg/api/v1/types.go

```diff
@@ -1352,6 +1352,16 @@ const (
 	PullIfNotPresent PullPolicy = "IfNotPresent"
 )
 
+// TerminationMessagePolicy describes how termination messages are retrieved from a container.
+type TerminationMessagePolicy string
+
+const (
+	// FallbackToLogsOnErrorTerminationMessage means that if the termination message file is empty, the most
+	// recent contents of the container logs will be used as the termination message if the container exits
+	// with an error.
+	FallbackToLogsOnErrorTerminationMessage TerminationMessagePolicy = "FallbackToLogsOnError"
```

Comment:
I admit I was trying to be clever - I didn't want to cause yet another new default field that would suddenly have to be set everywhere.  I agree that when we add policies we always add a default, but this might be the first time we've added a new string policy to change an existing field.  I also wanted to avoid setting this as long as possible so that it could be imposed by pod admission instead of on pod templates.

It would be more consistent to do so though, yes.